src/domain/event/EventUtils.ts

```diff
@@ -164,20 +173,10 @@ export const getGoogleLink = (
   locale: Language
 ): string => {
   const location = event.location;
-  const streetAddress = getLocalisedString(
-    (location && location.streetAddress) || {},
-    locale
-  );
-  const postalCode = (location && location.postalCode) || {};
-  const addressLocality = getLocalisedString(
-    (location && location.addressLocality) || {},
-    locale
-  );
-  const coordinates =
-    location && location.position
-      ? // Get coordinates for Google in correct order
-        [...location.position.coordinates].reverse()
-      : [];
+  const streetAddress = getLocalisedString(location?.streetAddress, locale);
+  const postalCode = location?.postalCode;
+  const addressLocality = getLocalisedString(location?.addressLocality, locale);
+  const coordinates = [...(location?.position?.coordinates || [])].reverse();
```

Comment:
Maybe this line doesn't need the spread?

..?..?.reverse() || []

Reply:
I think that reverse function also mutate the original array, not only return value. I used spread syntax to avoid unpleasant side effects

Reply:
yeah you are right :) didn't think of that, my bad!

then you could do:

location?.position?.coordinates?.slice().reverse()

but then  I think your solution is more readable

Reply:
So I don't think this line needs any changes

Reply:
This same line was used on 3 different functions. So I added a helper function to get event location fields

Reply:
Yeah nice. Btw, why are coordinates always reversed?

Reply:
LinkedEvents return coordinates in different order [lng, lat] than other services expect, so we have to change them to [lat, lng]

Reply:
ok 👍